src/foam/u2/view/OverlayActionListView.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'OverlayActionListView',
+  extends: 'foam.u2.View',
+
+  documentation: 'An overlay that presents a list of actions a user can take.',
+
+  requires: [
+    'foam.core.ConstantSlot',
+    'foam.u2.md.OverlayDropdown'
+  ],
+
+  imports: [
+    'ctrl'
+  ],
+
+  properties: [
+    {
+      type: 'Action[]',
```

Comment:
We still have to specify class: 'FObjectArray', class: 'FObjectProperty', class: 'Boolean', etc for properties at the moment.

In the future all you will have to specify is type: '', but that's not fully hooked up yet.

Reply:
I'll change it, but it seems to work anyway the way it is. Any idea why that would be?

Reply:
Because you essentially just had a regular Property axiom installed, instead of a more type specific Array property.  The base property still works as a property, it just doesn't do any of the extra functionality like type checking and adapting that the more specific property types do.  And if you're not relying on that actual functionality then it will all still work.

You can even specify properties as just a name:

```
properties: [
  'data'
]
```

This creates a plain property named data, which still works like a property.  We use things like
```
properties: [
  {
    class: 'Array'
    name: 'prop'
  }
]
```
Because Array property adds things like a useful default factory and type checking if you try to set it to a non-property.

Reply:
Got it, thank you!